CoffeeChat/Extensions/String.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  String.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 1/31/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    subscript(_ range: CountableRange<Int>) -> String {
+        let start = index(startIndex, offsetBy: max(0, range.lowerBound))
+        let end = index(start, offsetBy: min(self.count - range.lowerBound,
```

Comment:
I need to just remember this, but is this the line-wrapping style that we're using in Pear? I think we've done the other style in our other apps, and we should stay consistent across codebases. I may be wrong though!

```
let end = index(
    start, 
    offsetBy: min(
        self.count - range.lowerBound, range.upperBound - range.lowerBound
    )
)
```

Maybe the `offsetBy: ` is now short enough to be inlined.